lxde-base/menu-cache/menu-cache-1.1.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Library to create and utilize caches to speed up freedesktop application menus"
+HOMEPAGE="http://lxde.sourceforge.net/"
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxde/${PN}.git"
+else
+	SRC_URI="mirror://sourceforge/lxde/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~mips ~ppc ~x86 ~amd64-linux ~x86-linux"
+fi
+
+LICENSE="LGPL-2.1+"
+# ABI is v2. See Makefile.am
+SLOT="0/2"
+IUSE=""
+
+RDEPEND="
+	dev-libs/glib:2
+	x11-libs/libfm-extra:=
+"
+DEPEND="${RDEPEND}
+	sys-devel/gettext
+	virtual/pkgconfig
+"
+
+src_install() {
+	default
+	find "${ED}" \( -name "*.a" -o -name "*.la" \) -delete
```

Comment:
try and see if you can specify `--disable-static` to `econf` so you dont need to unnecesaarily build it

Reply:
I tried with --disable-static and the difference is that only the .a file is not installed, the .la file still is, so it doesn't disable static.